src/main/java/org/dataone/bookkeeper/resources/QuotasResource.java

```diff
@@ -235,19 +249,21 @@ public Quota retrieve(
     @Timed
     @PUT
     @RolesAllowed("CN=urn:node:CN,DC=dataone,DC=org")
```

Comment:
While the `@RolesAllowed` is not part of this commit, we'll need to remove this eventually and use `@PermitAll` so we can update quotas with the admin group subjects (CNs, permitted MNs, and select ORCIDs).